### PR TITLE
 Add first to TestElementWrapper

### PR DESCRIPTION
**Pull request by OFFLlNE**
Do I leave the stuff prettier did in or shall I remove those formatting changes/things?

### CODE REVIEW

**Comment by oliverviljamaa** [comment]
@OFFLlNE It did mess up the HTML code examples a bit, sadly, so reverting at least those would be preferred.

File: README.md [hunk 1]
```diff
@@ -222,6 +220,27 @@ it('has one transferwise link with corrext text on the right', () => {
 
 </details>
 
+#### `.first() => TestElementWrapper`
+
+Returns a [`TestElementWrapper`](#testelementwrapper-api) (for chaining) for first element.
```
Comment (oliverviljamaa) [inline README.md]:
```suggestion
Returns a [`TestElementWrapper`](#testelementwrapper-api) (for chaining) for the first element.
```

File: README.md [hunk 2]
```diff
@@ -222,6 +220,27 @@ it('has one transferwise link with corrext text on the right', () => {
...
+<details>
+  <summary>Example</summary>
+
+```html
+<button class="btn btn-primary">Balance</button>
+<button class="btn btn-primary">Bank transfer</button> <button class="btn btn-primary">Card</button>
+```
+
+```js
+it('has card as third button', () => {
```
Comment (oliverviljamaa) [inline README.md]:
```suggestion
it('has balance as first button', () => {
```

File: src/mount/TestElementWrapper/TestElementWrapper.js [hunk 3]
```diff
@@ -30,6 +30,11 @@ export default class TestElementWrapper {
     return elementToTestElementWrapper(foundElement);
   }
 
+  first() {
+    const elementAtIndex = this[angularElementSymbol][0];
```
Comment (oliverviljamaa) [inline src/mount/TestElementWrapper/TestElementWrapper.js]:
```suggestion
    const firstElement = this[angularElementSymbol][0];
```

File: src/mount/TestElementWrapper/TestElementWrapper.js [hunk 4]
```diff
@@ -30,6 +30,11 @@ export default class TestElementWrapper {
...
+    return elementToTestElementWrapper(elementAtIndex);
```
Comment (oliverviljamaa) [inline src/mount/TestElementWrapper/TestElementWrapper.js]:
```suggestion
    return elementToTestElementWrapper(firstElement);
```

File: README.md (hunk 2 above)
Comment (OFFLlNE) [reply README.md]: I guess `as the first`

**Review by oliverviljamaa — CHANGES_REQUESTED** [review]
I left a few suggestions and it would be great to revert the Prettier HTML changes, but otherwise, a great job!

**Review by oliverviljamaa — APPROVED** [review]

**Pull request merged**